app/components/Base/index.js

```diff
@@ -21,15 +27,23 @@ const Base = styled.div`
       ? `0 0 0 1px rgba(${BORDER_PRIMARY_RGB}, 0.4) !important`
       : `none`};
   border-top-left-radius: ${({ bordered, topRightRadius, withoutBR }) =>
-    (bordered || topRightRadius) && !withoutBR ? '5px' : 'none'};
+    (bordered || topRightRadius) && !withoutBR
```

Comment:
Can we find a way to reuse that expression instead of duplication?

Reply:
added three constants for border-radius (x, m, l sizes)